src/components/routes/Host/ListingForm/AccommodationsForm/AccommodationsForm.tsx

```diff
@@ -31,57 +32,61 @@ const AccommodationsForm = (props: any): JSX.Element => {
         </SelectBoxWrapper>
       </div> */}
 
-      <div className="form-item row">
-        <div className="input-container">
-          <InputLabel htmlFor="numberOfBedrooms">Bedrooms</InputLabel>
-          <InputWrapper>
-            <Field
-              name="numberOfBedrooms"
-              placeholder="# of Bedrooms"
-              type="number" />
-          </InputWrapper>
-          <ErrorMessageWrapper>
-            <ErrorMessage name="numberOfBedrooms" />
-          </ErrorMessageWrapper>
-        </div>
-
+      <div className="form-item">
         <div className="input-container">
-          <InputLabel htmlFor="sleepingArrangement">Beds</InputLabel>
+          <InputLabel htmlFor="sleepingArrangement">Sleeping Arrangement</InputLabel>
           <InputWrapper>
             <Field
               name="sleepingArrangement"
-              placeholder="# of Beds"
+              placeholder="1 King, 2 Queens"
               type="text" />
           </InputWrapper>
         </div>
       </div>
 
-      <div className="form-item row">
-        <div className="input-container">
-          <InputLabel htmlFor="numberOfBathrooms">Bathrooms</InputLabel>
-          <InputWrapper>
-            <Field
-              name="numberOfBathrooms"
-              placeholder="# of Bathrooms"
-              step={0.5}
-              type="number" />
-          </InputWrapper>
-          <ErrorMessageWrapper>
-            <ErrorMessage name="numberOfBathrooms" />
-          </ErrorMessageWrapper>
+      <div className="form-item">
+        <div className="input-number-container">
+          <InputLabel>Number of Bedrooms</InputLabel>
+          <NumberInput
+            value={values.numberOfBedrooms}
+            max={50}
+            min={1}
+            onChange={(value: number) => {
+              setFieldValue('numberOfBedrooms', value);
+              setFieldTouched('numberOfBedrooms');
+            }}
+          />
         </div>
-        
-        <div className="input-container">
-          <InputLabel htmlFor="sharedBathroom">Shared Bathroom</InputLabel>
-          <InputWrapper>
-            <Field
-              name="sharedBathroom"
-              placeholder="Yes or No"
-              type="text" />
-          </InputWrapper>
+      </div>
+
+      <div className="form-item">
+        <div className="input-number-container">
+          <InputLabel>Number of Bathrooms</InputLabel>
+          <NumberInput
+            value={values.numberOfBathrooms}
+            max={50}
+            min={0}
+            onChange={(value: number) => {
+              setFieldValue('numberOfBathrooms', value);
+              setFieldTouched('numberOfBathrooms');
+            }}
+            step={0.5}
+          />
         </div>
       </div>
 
+      <div className="form-item">
+        <Checkbox
+          checked={values.sharedBathroom === 'Yes' || values.sharedBathroom === 'yes'}
```

Comment:
why don't we use a boolean for this instead of a string?

Reply:
boolean would be ideal for `sharedBathroom`. I think it was a string from the beginning =X